### ISSUE TITLE
Getting error in Next.js app since yesterday

### ISSUE THREAD

**Issue by AndyPandyGeek** [MomenSherif/react-oauth#280]
Call Stack
_.$e
https://accounts.google.com/gsi/client (128:335)
new rp
https://accounts.google.com/gsi/client (227:3)
aq
https://accounts.google.com/gsi/client (246:34)
No
https://accounts.google.com/gsi/client (244:246)
Object.Oo [as renderButton]
https://accounts.google.com/gsi/client (216:62)
eval
node_modules/@react-oauth/google/dist/index.esm.js (91:0)

There's a quite similar issue opened yesterday: https://github.com/metabase/metabase/issues/32602

**Comment by AndyPandyGeek** [comment]
Issue solved by making the width property a number instead of string. However I had to disable type check (// @ts-nocheck), otherwise was getting an error that the module won't accept width type number.

**Comment by DevVanilli** [comment]
Thanks. That workaround works.

**Comment by eprothro** [comment]
Thanks for posting your workaround. Did not have to disable type check for our use, as current version types `GsiButtonConfiguration` with `width?: string | number`. Adding info to help others find via google:

Worked to resolve Google Identity Services (GSI) oauth login button render error.

**Comment by Rylab** [comment]
Thank you for the quick patch for this! Changing width from string to number confirmed as fixing this issue for us in a few apps, too. Thanks, Google 🙄

**Comment by kipyegonline** [comment]
Yes, It works, thanks alot.... the app was crashing... change width from string to number....
